packages/chez-scheme/build.sh

```diff
@@ -0,0 +1,50 @@
+TERMUX_PKG_HOMEPAGE=https://cisco.github.io/ChezScheme
+TERMUX_PKG_DESCRIPTION="Chez Scheme is both a programming language, an implementation, and a superset of R6RS"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="Komo @mbekkomo"
+TERMUX_PKG_VERSION=10.1.0
+TERMUX_PKG_SRCURL=https://github.com/cisco/ChezScheme/releases/download/v${TERMUX_PKG_VERSION}/csv${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9181a6c8c4ab5e5d32d879ff159d335a50d4f8b388611ae22a263e932c35398b
+TERMUX_PKG_DEPENDS="libiconv, liblz4, libx11, ncurses, zlib"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+# Copied from chez-scheme APKBUILD
+case ${TERMUX_ARCH} in
+x86_64) _chez_host=ta6le ;;
+aarch64) _chez_host=tarm64le ;;
+i686) _chez_host=ti3le ;;
+arm) _chez_host=tarm32le ;;
+esac
```

Comment:
Why isn't `_chez_host` exported in `termux_step_post_get_source()`?
If you need it by host build time it goes in there.

Also quote `${TERMUX_ARCH}` in the case line.